docs/model-dev-guide/create-experiment.rst

```diff
@@ -1,8 +1,8 @@
 .. _experiments:
 
-###################
- Submit Experiment
-###################
+#################################
+ Create and Submit an Experiment
```

Comment:
Just call it "Create an Experiment", there is no such thing as create without submitting.

Reply:
@tara-det-ai suggested - and I think it's good - that the use of both `create` and `submit` is helpful to a novice user who is _searching_ for information about building an experiment.